corehq/apps/app_manager/views/translations.py

```diff
@@ -35,22 +35,34 @@ def get_index_for_defaults(langs):
 @require_can_edit_apps
 @get_file("bulk_upload_file")
 def upload_bulk_ui_translations(request, domain, app_id):
+
+    def _html_message(header_text, messages):
+        message = _(header_text or "Upload failed. We found problem with following translations:")
```

Comment:
I think you need to have the translation call directly wrap the string in order for it to be flagged. not 100% sure about that though.